### PR TITLE
Feature/27 sql server dependency

### PR DESCRIPTION
**Pull request by shatl**
Move SqlExceptionDestructurer to Serilog.Exceptions.SqlServer assembly.

### CODE REVIEW

File: Source/Serilog.Exceptions.SqlServer/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,3 @@
+﻿using System.Runtime.CompilerServices;
```
Comment (RehanSaeed) [inline Source/Serilog.Exceptions.SqlServer/Properties/AssemblyInfo.cs]: Apparently there is a way to use `InternalsVisibleTo` in csproj. Someone commented in my blog post 'Cleaning up csproj'.

**Comment by shatl** [comment]
@RehanSaeed, any chances to get it merged soon?

**Comment by RehanSaeed** [comment]
Sorry, I've not been well for the last few days and have been trying to catch up. Will take a look at this in the next few days.

**Comment by shatl** [comment]
any updates ?

**Comment by RehanSaeed** [comment]
Thank you for your efforts. Will get a release out soon.

**Pull request merged**